.gitignore

```diff
@@ -16,10 +16,12 @@ package-lock.json
 databases/*.db*
 *.o
 *.o.dSYM
+.env
 # Typescript build artifacts
 .*-dist/
 tools/set-import/importer.js
 tools/modlog/converter.js
+**/sourceMaps
```

Comment:
Isn't this already covered by the `.*-dist` exclusion?

Reply:
`tools/set-import/importer.js` and `converter.js` are listed explicitly, so rather than repeat them, I just added a rule for **/sourceMaps.